apps/settings/test/unit/modules/addon_manager_test.js

```diff
@@ -80,29 +79,11 @@ suite('addons manager test > ', function() {
     origin: 'app://certifiedorigin'
   };
 
-  var MockContentScripts = {
-    addon1: [],
-    addon2: [{
-      matches: '<all_urls>',
-      js: [],
-      css: ['foo.css']
-    }],
-    privilegedAddon: [{
-      matches: '<all_urls>',
-      js: ['addon.js'],
-      css: []
-    }],
-    certifiedAddon: [{
-      matches: '<all_urls>',
-      js: ['addon.js'],
-      css: []
-    }]
-  };
-
   var addon1 = {
     manifest: {
       role: 'addon',
       name: 'testAddonName1',
+      content_scripts: []
```

Comment:
we should add note somewhere to notice us that the `content_scripts` field is internally copied from `manifest.json` to `manifest.webapp` in gecko